File: cercles.c
/*
cercles.c
Hugo MILLOT | Maël CHALON
Give stats about a circle that the user choose
*/
#include <stdio.h>
#define PI 3.14159

int main() {
    float radius;
    printf("Enter a radius : \n");
    scanf("%f",&radius);
    float area = radius * radius * PI;
    printf("Area of your circle is %f \n", area);
    float perimeter = 2 * PI * radius;
    printf("Perimeter of your circle is %f \n", perimeter);
}

File: cercles.py
# cercles.py
# Mael CHALON | Hugo MILLOT
# calculer l'aire et le perimètre d'un cerle dont le rayon est transmis par l'utilisateur


import math

radius:float = float(input("Enter the radius of the circle: ")) # input utilisateur pour les raidus

area:float = math.pi * radius * radius # calcul de l'aire

perimeter:float = math.pi * radius * 2 # calcul du perimettre

print(f"area = {area}, perimeter = {perimeter}") # affichage des resultats obtenus




